doc/installing.rst

```diff
@@ -1,42 +1,35 @@
 Installing joblib
 ===================
 
-The `easy_install` way
------------------------
+Using `pip`
+-----------
 
-For the easiest way to install joblib you need to have `setuptools`
-installed.
+The easiest way to install joblib is to use `pip`::
 
 * For installing for all users, you need to run::
 
-    easy_install joblib
+    pip install joblib
 
   You may need to run the above command as administrator
 
   On a unix environment, it is better to install outside of the hierarchy
   managed by the system::
 
-    easy_install --prefix /usr/local joblib
+    pip install --prefix /usr/local joblib
 
 * Installing only for a specific user is easy if you use Python 2.6 or
   above::
 
-    easy_install --user joblib
+    pip install --user joblib
 
-.. warning::
+Using `conda`
```

Comment:
indeed, I got confused with what was installed and the results of `conda search` ... Let's only keep pip then.